Makefile

```
@@ -4,7 +4,8 @@ TESTS_IMAGE=packit-tests
 CONTAINER_ENGINE ?= $(shell command -v podman 2> /dev/null || echo docker)
 TESTS_RECORDING_PATH=tests_recording
 TESTS_TARGET ?= ./tests/unit ./tests/integration ./tests/functional
-
+CONTAINER_COMMON=$(CONTAINER_ENGINE) run --rm -ti --env TESTS_TARGET=$(TESTS_RECORDING_PATH) -v $(CURDIR):/src --security-opt label=disable
+CONTAINER_WHAT=$(TESTS_IMAGE) bash -c "pip3 install .; make check"
```

Drop `--security-opt` and use proper SELinux labeling instead.

Change the name of these vars for better readbility.

```suggestion
CONTAINER_RUN=$(CONTAINER_ENGINE) run --rm -ti --env TESTS_TARGET=$(TESTS_RECORDING_PATH) -v $(CURDIR):/src:Z
CONTAINER_COMMAND=$(TESTS_IMAGE) bash -c "pip3 install .; make check"
```